src/main/services/auth.ts

```diff
@@ -17,8 +17,10 @@ export abstract class AuthService {
 
 @injectable()
 export class AuthServiceMock extends AuthService {
+    protected organisationId: string | null = null;
     async authorize(_ctx: Koa.Context) {}
-    getOrganisationId() { return null; }
+    getOrganisationId() { return this.organisationId; }
+    setOrganisationId(id: string) { this.organisationId = id }
```

Comment:
Yeah, def something's off with `semi` rule

Reply:
yup, it was off(probably one of the plug-in has it as off?) I've added it back 👍